lib/src/browser.dart

```diff
@@ -83,16 +83,16 @@ class _Chrome extends Browser {
 
   static bool _isChrome(NavigatorProvider navigator) {
     var vendor = navigator.vendor;
-    return vendor != null && vendor.contains('Google');
+    return vendor.contains('Google');
   }
 
   static Version _getVersion(NavigatorProvider navigator) {
-    Match match = RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
+    Match? match = RegExp(r"Chrome/(\d+)\.(\d+)\.(\d+)\.(\d+)\s")
         .firstMatch(navigator.appVersion);
     if (match != null) {
-      var major = int.parse(match.group(1));
-      var minor = int.parse(match.group(2));
-      var patch = int.parse(match.group(3));
+      var major = int.parse(match.group(1)!);
+      var minor = int.parse(match.group(2)!);
+      var patch = int.parse(match.group(3)!);
       var build = match.group(4);
```

Comment:
The `vendor` local var isn't providing much value anymore, could probably simplify this:

```dart
  static bool _isChrome(NavigatorProvider navigator) => vendor.contains('Google');
```

Reply:
This is on the wrong line, too.. the VS Code plugin must not work quite right :/